zubhub_frontend/zubhub/src/assets/js/styles/views/projects/projectsStyles.js

```diff
@@ -25,21 +25,32 @@ const styles = theme => ({
     // '& .MuiGrid-root.MuiGrid-container': {
     //   width: '100%',
     [theme.breakpoints.down('sm')]: {
-      padding: '0 24px',
+      // padding: '0 24px',
       width: '100%',
     },
   },
   heroMainSectionStyle: {
     backgroundColor: 'var(--primary-color1)',
     display: 'flex',
-    paddingTop: '2.3em',
-    paddingBottom: '2.8em',
+    padding: '0 16.5em',
+    paddingTop: '2em',
+    paddingBottom: '2em',
     justifyContent: 'center',
-    width: '100vw',
+    // width: '100%',
+    // maxWidth: '1230px',
     marginBottom: '2em',
     height:'100vh',
+    [theme.breakpoints.down('1620')]: {
```

Comment:
I don't feel adding 3 extra breakpoints - two of which are very close and very arbitrary is the correct way to solve this. Breakpoints should remain fixed and within them, the hero elements can size relatively.  For example. explore use of `em` for font sizing and for padding, there is some other underlying issue that padding is not adjusting on existing breakpoints. I am sure there is a much cleaner way to do this